### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by captainsafia**
Move helpful content from the [wiki](https://github.com/dotnet/aspnetcore/wiki/Blazor-Development-Sprints#how-to-submit-a-pr-and-expectations) to the contributing docs to make things more accessible.

cc: @BrennanConroy Part of the work towards making ASP.NET Core sprints happen.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -26,25 +26,48 @@ Our team members also monitor several other discussion forums:
...
   When you get an agreement from our team members that the design proposal you have is solid, then go ahead and prepare the PR.
   To file a design proposal, look for the relevant issue in the `New issue` page or simply click [this link](https://github.com/dotnet/aspnetcore/issues/new?assignees=&labels=design-proposal&template=4_design_proposal.md):
   ![image](https://user-images.githubusercontent.com/34246760/107969904-41b9ae80-6f65-11eb-8b84-d15e7d94753b.png)
   
-By following the guidance provided above, you will greatly increase chances for your PR to be accepted and merged.
+**Before submitting the pull request**
+
+Before submitting a pull request, make sure that it checks the following requirements:
+
+- You find an existing issue with the "help-wanted" label or discuss with the Blazor team to agree on adding a new issue with that label
```
Comment (BrennanConroy) [inline CONTRIBUTING.md:55]: ~~Blazor~~

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -26,25 +26,48 @@ Our team members also monitor several other discussion forums:
...
+- You post a high-level description of how it will be implemented, and receive a positive acknowledgement from the team before getting too committed to the approach or investing too much effort implementing it
+- You add test coverage following existing patterns within the codebase
+- Your code matches the existing syntax conventions within the codebase
+- Your PR is small, focused, and avoids making unrelated changes
+  
+If you're pull request contains any of the below, it's less likely to be merged.
```
Comment (BrennanConroy) [inline CONTRIBUTING.md:61]:
```suggestion
If your pull request contains any of the below, it's less likely to be merged.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -26,25 +26,48 @@ Our team members also monitor several other discussion forums:
...
+
+- Changes that break backward compatibility
+- Changes that are only wanted by one person/company. Changes need to benefit a large enough proportion of all Blazor developers.
```
Comment (BrennanConroy) [inline CONTRIBUTING.md:64]: ~~Blazor~~

**Review by mkArtakMSFT — APPROVED** [review]
Thanks Safia!

**Review by BrennanConroy — APPROVED** [review]

**Pull request merged**